testing/gearmand/gearmand.initd

```diff
@@ -0,0 +1,29 @@
+#!/sbin/runscript
+
+# Gearman server init.d file for alpine linux.
+
+name=gearmand
+daemon=/usr/sbin/$name
+
+depend() {
+	need net
+	after firewall
+}
+
+start() {
+	ebegin "Starting ${name}"
+		start-stop-daemon --start --quiet \
+			--pidfile /var/run/${name}.pid \
+			--chuid ${gearmand_user}:${gearmand_group} \
+			--exec ${daemon} -d ${gearmand_opts}
+	eend $?
+}
+
+stop() {
+	ebegin "Stopping ${name}"
+		start-stop-daemon --stop --quiet \
+			--pidfile /var/run/$name.pid \
+			--exec ${daemon}
+	eend $?
+}
```

Comment:
It’s unnecessary to implement start/stop functions. OpenRC is not stupid as SysV init and provides conventional variables for typical use cases. ;) You can write it witch much less boilerplate:

```sh
#!/sbin/runscript

name="gearmand"
description="Gearman server"

command="/usr/sbin/gearmand"
command_args="-d $gearmand_opts"
command_background="yes"

pidfile="/run/${RC_SVCNAME}.pid"
start_stop_daemon_args="--user=$gearmand_user:$gearmand_group"
```